cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+users you have to enable ACLs for these entities using the ``@Config`` annotation:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Entity/Task.php
+    namespace AppBundle\Entitiy;
+
+    use Oro\Bundle\EntityConfigBundle\Metadata\Annotation\Config;
+
+    /**
+     * @Config(
+     *     defaultValues={
+     *         "security"={
+     *             "type"="acl",
+     *             "group_name"=""
+     *         }
+     *     }
+     * )
+     */
+    class Task
+    {
+        // ...
+    }
+
+After you have done this and have cleared the cache you can toggle all kinds of permission checks
+(``CREATE``, ``EDIT``, ``DELETE``, ``VIEW``, and ``ASSIGN`` in the user role management interface.
+
+.. tip::
+
+    You can use the optional group attributes to create visual groups in the UI that can help the
+    user find your entity.
+
+.. _coobook-entities-acl-create:
+
+Creating Access Control Lists
+-----------------------------
+
+You have to options to define your custom access control lists:
+
+.. _cookbook-entity-acl-controller:
+
+#. In your controller class, you can use the ``@Acl`` annotation:
+
+   .. code-block:: php
+       :linenos:
+
+       // src/AppBunde/Controller/TaskController.php
+       namespace AppBundle\Controller;
+
+       use AppBunde\Entity\Task;
+       use Oro\Bundle\SecurityBundle\Annotation\Acl;
+       use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+       use Symfony\Component\HttpFoundation\Request;
+
+       class TaskController extends Controller
+       {
+           /**
+            * @Acl(
+            *   id="app_task_view",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="VIEW"
+            * )
+            */
+           public function indexAction()
+           {
+               // ...
+           }
+
+           /**
+            * @Acl(
+            *   id="app_task_create",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="CREATE"
+            * )
+            */
+           public function createAction(Request $request)
+           {
+               // ...
+           }
+
+           /**
+            * @Acl(
+            *   id="app_task_edit",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="EDIT"
+            * )
+            */
+           public function editAction(Task $task, Request $request)
+           {
+               // ...
+           }
+       }
+
+   Using the ``@Acl`` annotation does not only create new access control lists to which you can
+   refer in other parts of your code it will also trigger the access decision manager when your
+   actions are accessed by users and thus protect them from being accessed without the needed
+   permissions.
+
+#. If you do not want to protect any controller methods or if you prefer to keep the definition of
```

Comment:
Please add description that this definition have parameters `bindings` and `group_name`.
`group_name` the same is in Acl annotation. 
With `bindings` developer can specify controller/action where this definition will work. For example:

`
test_controller:
  type: entity
  class: AcmeBundle\Entity\SomeEntity
  permission: VIEW
  group_name: Test Group
  label: Test controller
  bindings:
    - { class: Acme\DemoBundle\Controller\TestController, method: testAction }
`